simtools/ray_tracing.py

```diff
@@ -307,7 +307,6 @@ def analyze(
                     containment_diameter_cm = image.get_psf(containment_fraction, "cm")
                     containment_diameter_deg = image.get_psf(containment_fraction, "deg")
                     centroid_x = image.centroid_x
```

Comment:
Why do you need to keep centroid_x?

Reply:
@GernotMaier did you mean the centroid_y? This one could be removed probably.

Reply:
I was wondering why we need only centroid x (and not centroid y). I can see that is used for the effective focal length calculation.

Doesn't that depend where in the FOV we put the simulated optical point source? I am wondering if this calculation is still correct when using another source position (isn't this a configurable option in the program and we can easily get a wrong effective focal length)

Reply:
And we can move this discussion also into an issue, as it is not related to the sonarqu reports.

Reply:
Isse is here #981